nss-day-cohort-29/reactive-nutshell-springwynne
# Styling

# Description

Styled the login, tasks and register pages 


## Type of change

- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

1. git fetch --all
1. git checkout "styling"
1. Navigate to register, login and through all task functionality to check out styling! Open to any and all notes 


# Checklist:

- [ X ] My code follows the style guidelines of this project
- [ X ] I have performed a self-review of my own code
- [ X ] My changes generate no new warnings
- [ X ] I have added test instructions that prove my fix is effective or that my feature works`
